belvedere/src/main/java/zendesk/belvedere/ImageStreamCursorProvider.java

```diff
@@ -0,0 +1,82 @@
+package zendesk.belvedere;
+
+import android.content.ContentResolver;
+import android.content.Context;
+import android.database.Cursor;
+import android.os.Build;
+import android.os.Bundle;
+import android.provider.MediaStore;
+
+import androidx.annotation.Nullable;
+import androidx.annotation.RequiresApi;
+
+import java.util.Locale;
+
+public class ImageStreamCursorProvider {
+
+    final static String[] PROJECTION = new String[]{
+            MediaStore.Images.ImageColumns._ID,
+            MediaStore.MediaColumns.DISPLAY_NAME,
+            MediaStore.MediaColumns.SIZE,
+            MediaStore.MediaColumns.WIDTH,
+            MediaStore.MediaColumns.HEIGHT
+    };
+
+    private final Context context;
+    private final BuildVersionProvider buildVersionProvider;
+
+    ImageStreamCursorProvider(Context context, BuildVersionProvider buildVersionProvider) {
+        this.context = context;
+        this.buildVersionProvider = buildVersionProvider;
+    }
+
+
+    @RequiresApi(api = Build.VERSION_CODES.Q)
```

Comment:
I'm always a little hazy on the meaning of these annotations so feel free to correct me, but I think we should not have this annotation here? From the docs:
```
Denotes that the annotated element should only be called on the given API level or higher. 
```

Given we explicitly cater for all API levels in the body of the method, I think this does not apply here. I always expect this annotation to be accompanied by an alternative code path for older versions. However as I said, I've never been 100% on what these annotations mean so please do correct me!

Reply:
I'll check - I was getting a compiler error without, but I'll see what we can do. I think there might have been a compiler error, but since we're handling both paths, it might be better to suppress.

Reply:
Was a compiler error, so I've suppressed the warnings since we're covering both paths